cypress/plugins/index.js

```diff
@@ -71,7 +71,7 @@ module.exports = (on, config) => {
   on('file:preprocessor', webpackPreprocessor(options));
 
   // eslint-disable-next-line global-require
-  require('cypress-terminal-report/src/installLogsPrinter')(on);
+  require('cypress-terminal-report/src/installLogsPrinter')(on, 10000);
```

Comment:
What does this number argument do? I think it's throwing an error in CI

Reply:
This was supposed to be the max limit of lines of log but I think I might have done this wrong. Will fix now

Reply:
(usually default is 800 which isn't enough)

Reply:
Ahh cool. Looks like you just need it as an object with the corresponding key
https://github.com/archfz/cypress-terminal-report#optionsdefaulttrimlength
```suggestion
  require('cypress-terminal-report/src/installLogsPrinter')(on, {defaultTrimLength: 10000});
```

Reply:
I have done this instead so we can add more options easily

 // Add options for the cypress terminal report (cy.logs) here
  const logPrinterOptions = {
    defaultTrimLength: 2000,
  };
  // eslint-disable-next-line global-require
  require('cypress-terminal-report/src/installLogsPrinter')(
    on,
    logPrinterOptions,
  );

E2Es pass now, unit test failed instead